api/.gitignore

```diff
@@ -5,7 +5,11 @@ venv
 __pycache__
 
 # run script
-run.sh
+run*.sh
```

Comment:
Now you can have a separate run-development.sh and run-production.sh or whatever if you want, and git will ignore it all.